docs/standard/whats-new/whats-new-in-dotnet-standard.md

```diff
@@ -71,7 +71,7 @@ If you install Visual Studio with the **.NET Core cross-platform development** w
 
 If you're using the .NET Core CLI, the following [dotnet new](../../core/tools/dotnet-new.md) command creates a class library project that targets the .NET Standard 2.0:
 
-```
+```console
```

Comment:
Shouldn't this be `dotnetcli` instead of `console` ?

Reply:
It should now that is supported

```suggestion
```dotnetcli
```

Reply:
Actually my PR yesterday already took care of this. Resolving the merge conflicts brought the expected identifiers in.